[workers] Update QR Code example (#17230)
cloudflare/cloudflare-docs: src/content/docs/workers/tutorials/build-a-qr-code-generator/index.mdx

## 3. Build a QR code generator

> All projects deployed to Cloudflare Workers support npm packages. This support makes it easy to rapidly build out functionality in your Workers. The [`qrcode-svg`](https://github.com/papnkukn/qrcode-svg) package is a great way to take text and encode it into a QR code. In the command line, install and save `qrcode-svg` to your project’s `package.json`:

Reviewer: @kristianfreeman , I think this suggestion is wrong. Sure, use single quotes when quoting text, but surely backticks are totally valid and preferred for inline code such as file and package names.

> All projects deployed to Cloudflare Workers support npm packages. This support makes it easy to rapidly build out functionality in your Workers. The [`qrcode-svg`](https://github.com/papnkukn/qrcode-svg) package is a great way to take text and encode it into a QR code. In the command line, install and save `qrcode-svg` to your project’s `package.json`:

Reviewer 2: Hey @bllchmbrs - are we able to tweak the style guide check for back ticks to allow it for things like files, package names, etc?

```sh title="Installing the qr-image package"
npm install --save qrcode-svg
```

In `index.js`, import the `qrcode-svg` package as the variable `QRCode`. In the `generateQRCode` function, parse the incoming request as JSON using `request.json`, and generate a new QR code using the `qrcode-svg` package. The QR code is generated as an SVG. Construct a new instance of `Response`, passing in the SVG data as the body, and a `Content-Type` header of `image/svg+xml`. This will allow browsers to properly parse the data coming back from your Worker as an image:

```js null {1,2,3,4,5,6}
import QRCode from "qrcode-svg";

async function generateQRCode(request) {
	const { text } = await request.json();
	const qr = new QRCode({ content: text || "https://workers.dev" });
	return new Response(qr.svg(), { headers: { "Content-Type": "image/svg+xml" } });
}
```

## 4. Test in an application UI

> The Worker will work if a user sends a `POST` request to a route, but it would be best practice to also be able to test the function with a proper interface. At this point in the tutorial, if any request is received by your function that is not a `POST`, a `405` response is returned. The new version of `fetch` should return a new `Response` with a static HTML document instead of the `405` error:

Reviewer 3: ```suggestion
The Worker will execute when a user sends a `POST` request to a route, but it is best practice to also provide a proper interface for testing the function. At this point in the tutorial, if any request is received by your function that is not a `POST`, a `405` response is returned. The new version of `fetch` should return a new `Response` with a static HTML document instead of the `405` error:
```
Slightly changing for grammatical clarity.

> In this tutorial, you built and deployed a Worker application for generating QR codes. If you would like to see the full source code for this application, you can find it [on GitHub](https://github.com/cloudflare/workers-sdk/tree/main/templates/examples/qr-code-generator).

Reviewer 3: @kristianfreeman This link is 404ing for me. I believe this folder was removed and the new path would be at https://github.com/cloudflare/workers-sdk/tree/main/packages/create-cloudflare but I'm not sure of the exact link for this one.
